android/capacitor/src/main/java/com/getcapacitor/Splash.java

```diff
@@ -41,8 +36,11 @@
 
   private static void buildViews(Context c) {
 
-    int splashId = c.getResources().getIdentifier("splash", "drawable", c.getPackageName());
-    Drawable splash = c.getResources().getDrawable(splashId);
+    // there are splashscreen creators naming the file sth other than "splash"
+    String splashResourceName = Config.getString("android.splashResourceName", "splash");
+
+    int splashId = c.getResources().getIdentifier(splashResourceName, "drawable", c.getPackageName());
+    Drawable splash = c.getResources().getDrawable(splashId, c.getTheme());
```

Comment:
is the `c.getTheme()` needed? the original code didn't have it.

Reply:
The previous used method `getDrawable(id)` is deprecated.

Reply:
Ah, cool. Looks good to me then